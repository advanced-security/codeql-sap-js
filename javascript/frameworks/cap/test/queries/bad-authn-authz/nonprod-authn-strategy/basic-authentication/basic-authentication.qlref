bad-authn-authz/NonProductionStrategyUsed.ql