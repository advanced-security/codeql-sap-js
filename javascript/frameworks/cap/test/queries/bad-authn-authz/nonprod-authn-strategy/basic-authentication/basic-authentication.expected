| package.json:16:17:16:23 | {\\n      ...       } | Non-production authentication strategy $@ is used. | package.json:16:17:16:23 | {\\n      ...       } | basic |
| package.json:17:18:32:9 | {\\n      ...       } | Current authentication strategy contains $@. | package.json:17:18:32:9 | {\\n      ...       } | credentials of mocked users |
